### ISSUE TITLE
A4A Error: Referrer Timeout!

### ISSUE THREAD

**Issue by calebcordry** [ampproject/amphtml#18288]
1700+ Errors in latest canary starting 9/22/18

```
Error: Referrer timeout!
    at Error (https://raw.githubusercontent.com/ampproject/amphtml/1809202142050/src/log.js:528:16)
    at apply (https://raw.githubusercontent.com/ampproject/amphtml/1809202142050/src/log.js:244:20)
    at apply (https://raw.githubusercontent.com/ampproject/amphtml/1809202142050/src/log.js:257:18)
    at error (https://raw.githubusercontent.com/ampproject/amphtml/1809202142050/ads/google/a4a/utils.js:254:8)
```

Stack trace appears to be a bit off.    Real error is generated here https://github.com/ampproject/amphtml/blob/146cde48394b09fe89da72bd696690acc9b6a554/ads/google/a4a/utils.js#L254

**Comment by calebcordry** [comment]
cc/ @lannka

**Comment by lannka** [comment]
looked at #18002 , seems this error message is expected. 

Should we change it to `expectedError`? @keithwrightbos 

downgrade to P2

**Comment by keithwrightbos** [comment]
Hmmm... do we have any idea how often this occurs?  I'll create a PR that wraps this in an experiment so I can determine how often the delay occurs and pick an optimal value

**Comment by calebcordry** [comment]
Closed by #18293
